src/Symfony/Component/Lock/Tests/Store/MongoDbStoreFactoryTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Lock\Tests\Store;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Lock\Store\MongoDbStore;
+use Symfony\Component\Lock\Store\StoreFactory;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+class MongoDbStoreFactoryTest extends TestCase
+{
+    public function testCreateMongoDbCollectionStore()
+    {
+        if (!class_exists(\MongoDB\Collection::class)) {
+            $this->markTestSkipped();
```

Comment:
can you try replacing all those by some `@required extension foo` annotation?

Reply:
Definitely! I was not aware of this capability of PHPUnit. Thanks for teaching me this!